server/app.js

```diff
@@ -29,22 +29,28 @@ const io = socketio(server, {
     origin: "*",
   },
 });
+app.set('socketio', io);
+io.on("connection",(socket) => {
+   
+  socket.on('JoinConversation', async({ userId,conversationId},cb) => {
+
+   const{error}=await addUser({id:socket.id,userId,conversationId})
```

Comment:
Eric, please format the code properly. This has been happening a lot in your PRs and you haven't been keeping consistent with style in our codebase. 

```suggestion
   const { error }= await addUser({ id: socket.id, userId, conversationId })
```